tests/package.json

```diff
@@ -12,7 +11,6 @@
     "drool": "0.2.2"
   },
   "scripts": {
-    "serve": "http-server -p 8000 ..",
```

Comment:
Well, the sort of nice thing about `gulp test-server` is that one doesn't have to be in the `tests` directory when executing that command. So we can say "Start the test server using `gulp test-server`" instead of "Start the test server by navigating to `tests` and running `npm run serve`". Plus, if we have `npm run serve` as an alias like that, it adds a small extra step in the chain that someone has to follow if they want to see what's happening with that command. For these minor reasons, I favor having `gulp test-server` at least be the primary command for running the test server (i.e. the command that is provided in README.md). However, if you are thinking more along the lines of leaving `npm run serve` as an undocumented legacy command in order to not break the API for people who have already become accustomed to using that command, then I'm down with that and would be happy to squash that change into this pull request.